src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,9 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  console.log(employeeName);
+
+  const found = employees.find((employee) => employee.name === employeeName);
+  console.log(found);
+  return found.role;
+};
```

Comment:
This could be reduced down a bit without changing the logic at all. As above, always remove the console logs once youre done with them :)
```
const getEmployerRole = (employeeName, employees) =>
  employees.find((employee) => employee.name === employeeName).role;
```